DigitalOffice/Tests/AuthenticationServiceTests/Validators/UserLoginValidatorTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using FluentValidation;
+using FluentValidation.TestHelper;
+using LT.DigitalOffice.AuthenticationService.Models;
+using LT.DigitalOffice.AuthenticationService.Validators;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.AuthenticationServiceUnitTests.Validators
+{
+    class UserLoginValidatorTests
+    {
+        private IValidator<UserLoginCredentialsRequest> validator;
+
+        [SetUp]
+        public void SetUp()
+        {
+            validator = new UserLoginValidator();
+        }
+
+        [Test]
+        public void EmptyLoginEmptyPassword()
+        {
+            validator.ShouldHaveValidationErrorFor(x => x.Email, "");
+            validator.ShouldHaveValidationErrorFor(x => x.Password, "");
+        }
+
+        [Test]
+        public void EmptyLogin()
+        {
+            validator.ShouldHaveValidationErrorFor(x => x.Email, "");
+            validator.ShouldNotHaveValidationErrorFor(x => x.Password, "Example");
```

Comment:
Why do we have password field here? I think that we can check one field in one test.